libs/sdk-backend-bear/src/convertors/toBackend/DashboardConverter.ts

```diff
@@ -74,56 +72,95 @@ const refToUri = (ref: ObjRef) => {
     );
 };
 
-const convertLayoutColumn = (
-    column: IFluidLayoutColumn | IFluidLayoutColumnDefinition,
-): GdcDashboardLayout.IFluidLayoutColumn => {
+const convertResponsiveSize = (size: IFluidLayoutSize): GdcDashboardLayout.IFluidLayoutSize => {
+    const converted: GdcDashboardLayout.IFluidLayoutSize = {
+        width: size.widthAsGridColumnsCount,
+    };
+    if (size.heightAsRatio) {
+        converted.heightAsRatio = size.heightAsRatio;
+    }
+
+    return converted;
+};
+
+const convertLayoutColumnSize = (
+    column: IFluidLayoutSizeByScreen,
+): GdcDashboardLayout.IFluidLayoutColSize => {
+    const allScreens: ResponsiveScreenType[] = ["xl", "md", "lg", "sm", "xs"];
+    return allScreens.reduce((acc: GdcDashboardLayout.IFluidLayoutColSize, el) => {
+        const size = column[el];
+        if (size) {
+            return {
+                ...acc,
```

Comment:
Nitpick: there is no need to spread here, you can directly mutate, the object was created by you and never leaves this scope before being "done".